pkgs/by-name/ck/ckb/package.nix

```
@@ -0,0 +1,56 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  ckb,
+  openssl,
+  pkg-config,
+  rustPlatform,
+  stdenv,
+  versionCheckHook,
+  testers,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "ckb";
+  version = "0.202.0";
+
+  src = fetchFromGitHub {
+    owner = "nervosnetwork";
+    repo = "ckb";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-LE+w5RHkDk+1Zu2RZgPyHbHJ92C1oAxGEcVc3Qo2lQc=";
+  };
+
+  cargoHash = "sha256-R7XV//e6n6afbad7HgwlcNwQRdO0xFZ0UGy1SU294Fs=";
+
+  buildFeatures = lib.optionals (stdenv.hostPlatform.isDarwin) [ "portable" ];
```

No need to have parens around a single value:

```suggestion
  buildFeatures = lib.optionals stdenv.hostPlatform.isDarwin [ "portable" ];
```

Personally, I'd format this with a multiline list, in anticipation of additional build features being specified in the future:


```suggestion
  buildFeatures = lib.optionals stdenv.hostPlatform.isDarwin [
    "portable"
  ];
```

But these are both nits (and/or preferences), and aren't critical.